ansible/roles/mesh-mgr/tasks/traefik.yml

```diff
@@ -0,0 +1,23 @@
+# Requires ansible-galaxy collection install kubernetes.core
+- name: Add Traefik repo
+  kubernetes.core.helm_repository:
+    name: stable
+    repo_url: "https://kubernetes.github.io/ingress-nginx"
+
+
+- name: Deploy Traefik 
+  kubernetes.core.helm:
+    name: traefik 
+    chart_ref: traefik/traefik 
```

Comment:
It comes installed by default, adding may cause problems

Reply:
Forgot about that, thanks.